site/smart-accounts/modular-account/multisig-plugin/usage-guide.md

```diff
@@ -0,0 +1,128 @@
+  - - meta
+    - property: og:description
+      content: Getting started with the Modular Account with Multisig Plugin in Account Kit
+  - - meta
+    - name: twitter:title
+      content: Modular Account with Multisig Plugin • Getting started
+  - - meta
+    - name: twitter:description
+      content: Getting started with the Modular Account with Multisig Plugin in Account Kit
+---
+
+# Getting started with Modular Account with Multisig Plugin
+
+To start, follow the guide to set up Modular Account [here](../getting-started.md).
+
+## Modular Account with Multisig Plugin Actions
+
+### Creating a Account Client
+
+The first step is to create a Multisig Modular Account client 
+
+```ts
+import { LocalAccountSigner } from "@alchemy/aa-core";
+import { createMultisigAccountAlchemyClient } from "@alchemy/aa-alchemy";
+
+// Creating a 3/3 multisig account 
+const signers = [LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 0 }
+  ), LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 1 }
+  ), LocalAccountSigner.mnemonicToAccountSigner(
+    MODULAR_MULTISIG_ACCOUNT_OWNER_MNEMONIC,
+    { accountIndex: 2 }
+  )];
+
+const threshold = 3n;
+
+const owners = signers.map(async(s) => await s.getAddress())
+
+const multisigAccountClient = createMultisigAccountAlchemyClient({
+    chain,
+    signers[0], 
+    owners,
+    threshold,
+    apiKey: API_KEY!
+  });
+```
+
+### Proposing a User Operation
+
+The first step to creating and sending a multisig User Operation is the propose step. This performs gas estimates, constructs the user operation struct, and generates a signature using the pre-provided signer.
+
+```ts
+import { createMultisigAccountAlchemyClient } from "@alchemy/aa-alchemy";
+
+const multisigAccountClient = createMultisigAccountAlchemyClient({
+    chain,
+    signers[0],  // using the first signer
+    owners,
+    threshold,
+    apiKey: API_KEY!,
+    gasManagerConfig: {
+      policyId: GAS_MANAGER_POLICY_ID,
+    }
+  });
+
+const { request, aggregatedSignature } = await multisigAccountClient.proposeUserOperation({
+    uo: {
+      target: targetAddress,
+      data: "0x",
+    }
+  });
+```
+
+### Getting the next signatures
+
+The next step is to collect the next k-2 signatures (excluding the first and the last signature). This is done with the `signMultisigUserOperation` method.
+
+```ts
+import { createMultisigAccountAlchemyClient } from "@alchemy/aa-alchemy";
+
+const multisigAccountClient = createMultisigAccountAlchemyClient({
+    chain,
+    signers[1], // using the second signer
+    owners,
+    threshold,
+    apiKey: API_KEY!
+  });
+
+const { aggregatedSignature } = await multisigAccountClient.signMultisigUserOperation({
+  account: multisigAccountClient.account,
+  signatures: [aggregatedSignature],
+  userOperationRequest: request,
+});
```

Comment:
I get what you mean with this sample code, but it's confusing that `aggregatedSignature` appears on both the left and right sides of this assignment and refers to different things. I'd suggest one or both of the following:

1. Change
  ```ts
   const { aggregatedSignature } =
   ```
   to
   ```ts
   const { aggregatedSignature: newAggregatedSignature } =
   ```
   or if you think that would be confusing to readers who aren't familiar with JavaScript's destructuring assignment syntax, you could do
   ```
   const newAggregatedSignature
   ```
   and put `.aggregatedSignature` at the very end of the right-hand-side.

2. In the right-hand-side, change `[aggregatedSignature]` to `[previousAggregatedSignature]` or something similar.

Reply:
good callout!